### DISCUSSION TITLE
怎么在节点上查询某个账户最近几天的交易？

### DISCUSSION THREAD

**Discussion by dafo374** [filecoin-project/community-china#75] [新手上路]
请教，如何用lotus指令查询某个账户最近的交易信息

**Comment by maxvint** [comment]
这个目前无法通过lotus命令行查询，需要调用Lotus的JSON RPC API，把交易数据同步到数据库中，进行检索。

**Comment by luozhanxin** [comment]
> 请教，如何用lotus指令查询某个账户最近的交易信息

这个得全节点吧？

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308
